app/Models/Song.php

```diff
@@ -183,9 +185,12 @@ public function updateSingle($title, $albumName, $artistName, $lyrics, $track, $
             $album = Album::get(Artist::getVarious(), $albumName, true);
         }
 
+        $this->title = $title;
```

Comment:
@phanan This ALSO fixes a bug - apparently there was an update that removed the song `title` from ever getting updated. This bug exists in master.